<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLViewer</class>
 <widget class="QMainWindow" name="PCLViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>806</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>640</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="nonMaxSuppBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="value">
     <double>4.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="nonMaxSupp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>nonMaxSuppMult</string>
    </property>
    <property name="buddy">
     <cstring>nonMaxSuppBox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="salientRadius">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>salRadMult</string>
    </property>
    <property name="buddy">
     <cstring>salientRadiusBox</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="salientRadiusBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="value">
     <double>6.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="normalRadius">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>normRadMult</string>
    </property>
    <property name="buddy">
     <cstring>normalRadiusBox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="borderRadius">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>bordRadMult</string>
    </property>
    <property name="buddy">
     <cstring>borderRadiusBox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="gamma21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>gamma21</string>
    </property>
    <property name="buddy">
     <cstring>gamma21Box</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="gamma32">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>gamma32</string>
    </property>
    <property name="buddy">
     <cstring>gamma32Box</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="normalRadiusBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="value">
     <double>4.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="borderRadiusBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="gamma21Box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.975000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="gamma32Box">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.975000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="ModelResolutionDisp">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="modelResolution">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ModelResolution</string>
    </property>
   </widget>
   <widget class="QLabel" name="nonMaxSupp_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>nonMaxSupp</string>
    </property>
    <property name="buddy">
     <cstring>nonMaxSuppBox_2</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="salientRadius_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>salientRadius</string>
    </property>
    <property name="buddy">
     <cstring>salientRadiusBox_2</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="normalRadius_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>normalRadius</string>
    </property>
    <property name="buddy">
     <cstring>normalRadiusBox_2</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="borderRadius_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>borderRadius</string>
    </property>
    <property name="buddy">
     <cstring>borderRadiusBox_2</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="nonMaxSuppBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="salientRadiusBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="normalRadiusBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>590</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="borderRadiusBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculateKeypoints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate 
Keypoints</string>
    </property>
   </widget>
   <widget class="QLabel" name="CubeEdgeLength">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CubeEdgeLength</string>
    </property>
    <property name="buddy">
     <cstring>gamma21Box</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="CubeEdgePoints">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CubeEdgePoints</string>
    </property>
    <property name="buddy">
     <cstring>gamma21Box</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="CubeEdgeLengthBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.400000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="CubeEdgePointsBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultValues">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>640</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default
Values</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MultiplyResolution">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MultiplyResolution</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="minNeighbors">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;MinNeighborsfor&lt;/p&gt;&lt;p&gt;NonMaxSupp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="buddy">
     <cstring>gamma21Box</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minNeighborsBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>30.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputModelBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Give a valid .pcd file to load the model from</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>PCD Input File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="UsePcdInputBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use PCD Input File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DownSampleBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Downsample</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="downSampleLeafSizeBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.010000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nonMaxSuppBox</tabstop>
  <tabstop>salientRadiusBox</tabstop>
  <tabstop>normalRadiusBox</tabstop>
  <tabstop>borderRadiusBox</tabstop>
  <tabstop>gamma21Box</tabstop>
  <tabstop>gamma32Box</tabstop>
  <tabstop>nonMaxSuppBox_2</tabstop>
  <tabstop>salientRadiusBox_2</tabstop>
  <tabstop>normalRadiusBox_2</tabstop>
  <tabstop>borderRadiusBox_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
